OmniKit/MessageTransport/MessageBlocks/GetStatusCommand.swift

```diff
@@ -9,29 +9,29 @@
 import Foundation
 
 public struct GetStatusCommand : MessageBlock {
-    
-//    public enum StatusType: UInt8 {
-//        case normal = 0x00
-//        case configuredAlerts = 0x01
-//        case faultEvents = 0x02
-//        case dataLog = 0x03
-//        case faultDataInitializationTime = 0x04
-//        case hardcodedValues = 0x06
-//        case resetStatus = 0x46 // including state, initialization time, any faults
-//        case dumpRecentFlashLog = 0x50
-//        case dumpOlderFlashlog  = 0x51 // but dumps entries before the last 50
-//        // https://github.com/openaps/openomni/wiki/Command-0E-Status-Request
-//    }
-    
+    // https://github.com/openaps/openomni/wiki/Command-0E-Status-Request
+
     public let blockType: MessageBlockType = .getStatus
     public let length: UInt8 = 1
     public let podInfoType: PodInfoResponseSubType
+
+//    public enum PodInfoResponseSubType: UInt8, Equatable {
+//        case normal                      = 0x00
```

Comment:
Having this here is likely to get out of date with the actual definition in PodInfo.swift

Reply:
Just trying to update stuff that was already there. I'm happy to just toss it.